src/v2/Apps/Artists/ArtistsApp.tsx

```diff
@@ -0,0 +1,270 @@
+
+  return (
+    <>
+      <AppContainer>
+        <HorizontalPadding>
+          <Box
+            display="flex"
+            flexDirection={["column", "row"]}
+            justifyContent="space-between"
+            alignItems="center"
+            borderTop="1px solid"
+            borderBottom="1px solid"
+            borderColor="black10"
+            py={2}
+            px={[2, 0]}
+            my={3}
+          >
+            <Text>Browse over 100,000 artists</Text>
+
+            <Box display="flex" flexWrap="wrap" justifyContent="center">
+              {LETTERS.map(letter => {
+                return (
+                  <Text>
+                    <RouterLink
+                      key={letter}
+                      to={`/artists/${letter}`}
+                      style={{ padding: space(0.5), textDecoration: "none" }}
+                      title={`View artists starting with “${letter}”`}
+                    >
+                      {letter}
+                    </RouterLink>
+                  </Text>
+                )
+              })}
+            </Box>
+          </Box>
+
+          <Text as="h1" variant="largeTitle" my={2} textAlign="center">
+            {headline}
+          </Text>
+
+          <Carousel my={2} arrowHeight={410}>
+            {artists.map((link, i) => {
+              const { image } = link
+
+              if (!image) return null
+
+              return (
+                <RouterLink
+                  key={link.internalID}
+                  to={link.href}
+                  style={{ display: "block", textDecoration: "none" }}
+                  aria-label={link.title}
+                >
+                  <Image
+                    src={image.thumb.src}
+                    srcSet={image.thumb.srcSet}
+                    width={image.thumb.width}
+                    height={image.thumb.height}
+                    alt={link.title}
+                    lazyLoad={i > 2}
+                  />
+
+                  <Box my={1} display="flex" justifyContent="space-between">
+                    <Box>
+                      <Text as="h3" variant="title">
+                        {link.title}
+                      </Text>
+
+                      <Text as="h4" variant="subtitle">
+                        {link.subtitle}
+                      </Text>
+                    </Box>
+
+                    <FollowArtistButtonQueryRenderer
+                      id={link.internalID}
+                      contextModule={ContextModule.featuredArtistsRail}
+                    />
+                  </Box>
+                </RouterLink>
+              )
+            })}
+          </Carousel>
+
+          {genes.map(gene => {
+            if (gene.trendingArtists.length === 0) return null
+
+            return (
+              <Box key={gene.name} my={3}>
+                <Text as="h2" variant="subtitle" my={1}>
+                  {gene.name}
+                </Text>
+
+                <CSSGrid
```

Comment:
(nonblocking) Maybe use your new grid component recently added to palette? Thats a little friendlier than having to grok the CSSGrid api